core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -156,10 +154,12 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
                     icon: 'pi-ellipsis-v',
                     label: 'editema.editor.navbar.properties',
                     action: () => {
-                        this.store.initActionEdit({
-                            inode: page.inode,
-                            title: page.title,
-                            type: 'shell'
+                        this.dialog.editContentlet({
+                            contentlet: {
+                                inode: page.inode,
+                                title: page.title,
+                                identifier: page.identifier
```

Comment:
Is this being tested?

Reply:
No because this triggers inside another component I'm not really sure how to test this or if we need to test it.

Reply:
Arent we testing this in the shell tests? meaning like when click on the sidenav or something? or at least are we testing what we pass to the nav?

Reply:
No test whatsoever. I will add tests that we are passing this items to the nav bar.